packages/oui-navbar/_mixins.less

```diff
@@ -6,136 +6,461 @@
+      position: relative;
+      padding: @menu-padding;
+
+      @media screen and (max-width: 1024px) {
+        display: block;
+      }
+    }
+
+    // Title for Header
+    &-title {
+      line-height: @menu-title-line-height;
+      margin: 0;
       padding: 0;
-      list-style: none;
-      white-space: nowrap;
+      text-transform: uppercase;
+      font-size: @menu-title-font-size;
+      font-weight: bold;
+      color: @menu-title-font-color;
     }
 
-    &__item {
-      float: left;
+    // Bradcrumb for Header
+    &-breadcrumb {
       display: block;
-      margin-left: rem-calc(16);
-      border-radius: 2px;
+      margin: 0;
+      font-size: @menu-font-size_small;
+    }
+
+    // Back Button
+    &-back {
+      position: absolute;
+      left: @menu-padding;
+      top: 50%;
+      transform: translateY(-50%);
+
+      .oui-button,
+      .oui-icon {
+        padding: 0;
+        margin-bottom: 0;
+      }
+    }
+
+    &-back ~ &-title,
+    &-back ~ &-breadcrumb {
+      padding-left: @menu-back-padding;
+    }
+  }
+
+  .navbar-menu-fixed(
+    @height: @oui-navbar-height
+  ) {
+    overflow-x: hidden;
+    overflow-y: auto;
+    position: fixed;
+    top: @height;
+    bottom: 0;
+
+    & & {
+      top: 0;
+    }
+  }
+
+  .navbar-menu-animation_opacity() {
+    opacity: 0;
+    visibility: hidden; // For ARIA
+    pointer-events: none;
+    transition: opacity 0.2s ease-out;
+
+    [aria-expanded="true"] ~ & {
+      opacity: 1;
+      visibility: visible;
+      pointer-events: auto;
+      transition: opacity 0.2s ease-out, visibility 0;
+    }
+  }
+
+  .navbar-menu-animation_slide() {
+    opacity: 1;
+    visibility: hidden; // For ARIA
+    pointer-events: auto;
+    transform: translateX(-100%);
+    transition: transform 0.25s ease-out, visibility 0.25s;
+
+    &_right {
+      transform: translateX(100%);
     }
 
-    &__item_current,
-    &__item:hover {
-      background-color: @oui-color-fiord;
+    [aria-expanded="true"] ~ & {
+      visibility: visible;
+      transform: translateX(0);
+      transition: transform 0.25s ease-out, visibility 0;
     }
+  }
+
+  .navbar-toggler(
+    @background-color_active: @oui-navbar-menu-background-color,
+    @font-color: @oui-navbar-font-color,
+    @font-color_active: @oui-navbar-menu-font-color,
+    @font-size: @oui-navbar-font-size,
+    @icon-size: @oui-navbar-icon-size,
+    @icon-bar-size: @oui-navbar-icon-bar-size
+  ) {
+    #oui > .navbar-link();
 
-    &__link {
+    padding: 0;
+    display: none;
+    text-align: center;
+
+    @media screen and (max-width: 1024px) {
       display: block;
-      padding: rem-calc(7) rem-calc(16);
-      border: 0;
-      background: none;
-      overflow: hidden;
-      font-size: rem-calc(14);
-      font-weight: 600;
-      text-decoration: none;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      cursor: pointer;
     }
 
-    .@{select-selector} {
-      margin-bottom: 0;
+    &-icon {
```

Comment:
(optional) Maybe just define that it defines the hamburger icon ;)

Reply:
Yes :)